config/config.go

```diff
@@ -388,6 +388,17 @@ type Local struct {
 	// BlockServiceCustomFallbackEndpoints is empty.
 	// The archiver is randomly selected, if none is available, will return StatusNotFound (404).
 	EnableBlockServiceFallbackToArchiver bool `version[16]:"true"`
+
+	// CatchupBlockValidateMode defines the degree of validation used by the catchup service. This field is a bit-field, where the
+	// default enabled checks have their corresponding bit set as zero, and the default disabled checks have their corresponding bit clear.
+	// bit 0, default enable: verify the block certificate
+	// bit 1, default enable: verify payset committed hash in block header matches payset hash
+	// bit 2, default disable: verify the transaction signatures on the block are valid
+	// bit 3, default disable: verify that the hash of the recomputed payset matches the hash of the payset committed in the block header.
+	// not all permutation of the above bitset are currently functional. In particular, the two that are functional are:
+	// 0 : default behavior.
+	// 12 : perform all supported validation methods.
```

Comment:
Isn't 12 the default position?
i.e. only CatchupVerifyCertificate and CatchupVerifyPaysetHash return true?

Reply:
I see the formulas are different for the upper bits. This is correct.